.github/CODEOWNERS

```diff
@@ -0,0 +1,52 @@
+###################################################################################################
+# Overview
+#
+# Pattern used to match files follows most of the same rules as used in gitignore files. Order is
+# important; the last matching pattern takes precendence.
+#
+# For more info see:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+###################################################################################################
+
+
+###################################################################################################
+# Global (@keycloak/maintainers)
+###################################################################################################
+
+*                                                           @keycloak/maintainers
+
+###################################################################################################
+# Testing (@keycloak/testing-maintainers)
+###################################################################################################
+
+/testsuite/                                                 @keycloak/testing-maintainers
```

Comment:
I'm not quite sure about having a separate team for `testing-maintainers`. Isn't technically every member of a `-mainstainers` team a testsuite maintainer?

Do we plan to have folks that are only testsuite maintainers but not actually maintainers of the component that the test validates?

I'm sorry if I missed some conversation about it.

Reply:
The point of this team is to avoid requiring a global maintainer to approve each pull request containing tests (which should be the majority of PRs). It was discussed in the google doc.

Reply:
Thanks @mhajas. Yes, that makes sense. My point though was that each team maintainer is implicitly also a testing maintainer. Therefore I believe we don't need this team. Or am I wrong?

Reply:
Not sure I understand what you mean. Example of the way I understand this: if we don't include this line and there is a change in the `testsuite/*` it doesn't match any other `*-maintainers` team therefore it will match only the first `*` and require a global maintainer to review. With this line and every `*-maintainer` in that group, it won't be necessary.

btw. @stianst can we avoid creating a new group for testing-maintainers and change this line to the following or we don't want this?
```suggestion
/testsuite/                                                 @keycloak/cloud-native-maintainers @keycloak/store-maintainers @keycloak/ui-maintainers
```

Reply:
Found the conversation. I understand now it is a technical limitation. Then forget my comment. :)